diff -ur alpaka-1.2.0-orig/cmake/alpakaCommon.cmake alpaka-1.2.0-fixed/cmake/alpakaCommon.cmake
--- alpaka-1.2.0-orig/cmake/alpakaCommon.cmake	2024-10-02 08:27:25.000000000 +0200
+++ alpaka-1.2.0-fixed/cmake/alpakaCommon.cmake	2025-05-13 10:36:55.325318647 +0200
@@ -214,6 +214,9 @@
     SET(Boost_DETAILED_FAILURE_MSG ON)
 endif()
 
+if(POLICY CMP0167)
+    cmake_policy(SET CMP0167 OLD)
+endif()
 find_package(Boost ${_alpaka_BOOST_MIN_VER} REQUIRED
              OPTIONAL_COMPONENTS atomic)
 
